sensor_msgs/RegionOfInterest[] roi_array